mgmt/smp/pkg.yml

```diff
@@ -18,7 +18,7 @@
 #
 
 pkg.name: mgmt/smp
-pkg.description: Server-side SMP functionality.
+pkg.description: Server-side Simple Management Protocol functionality.
```

Comment:
SMP was pre-chosen so I think we should keep it. You should just mention it in parenthesis.